share/completions/apm.fish

```diff
@@ -18,21 +18,17 @@ function __fish_apm_needs_command
     return 1
 end
 
-# Returns exit code of 0 if a command (argv[1]) appears once,  exclusions can
-# can be provided (argv[2..-1]) that are ignored, e.g. `get` and `set`
+# Returns exit code of 0 if any command (argv[1..-1]) appears once, ignores flags.
 function __fish_apm_using_command
-    set command $argv[1]
-    if test (count $argv) -gt 1
-        set exclusions $argv[2..-1]
-    end
+    set commands $argv[1..-1]
```

Comment:
The slice is actually unnecessary - it'll always expand to the full array anyway.